src/scripts/main.js

```diff
@@ -1,3 +1,189 @@
 'use strict';
 
-// write code here
+function parseSalary(sum) {
+  return +sum.slice(1).split(',').join('');
+}
+
+function formatToSalary(num) {
+  return `$${num.toLocaleString('en-US')}`;
+}
+
+function pushNotification(title, description, type) {
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification', type);
+  notification.setAttribute('data-qa', 'notification');
+
+  notification.innerHTML = `
+    <h2 class="title">${title}</h2>
+    <p>${description}</p>
+  `;
+
+  document.body.append(notification);
+
+  setTimeout(() => {
+    notification.remove();
+  }, 3000);
+};
+
+const table = document.querySelector('table');
+const body = document.querySelector('tbody');
+
+let activeHeaderIndex = null;
+let selectedRow = null;
+
+// Sorting by clicking on title
+table.addEventListener('click', (e) => {
+  if (e.target.tagName !== 'TH') {
+    return;
+  }
+
+  const rows = [...document.querySelectorAll('tbody>tr')];
+  const index = e.target.cellIndex;
+
+  if (activeHeaderIndex !== index) {
+    activeHeaderIndex = index;
+
+    rows.sort((a, b) => {
+      const aText = a.children[index].textContent;
+      const bText = b.children[index].textContent;
+
+      if (index === 4) {
```

Comment:
what means number 4 in this case? let's create obvious variables for column indices